js/application.js

```diff
@@ -126,194 +209,507 @@ $(document).ready(function(){
+        <div id="column5" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column6" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column7" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column8" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+      </div>\
+      <div class = "row" id="grid_6">\
+        <div id="column1" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column2" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column3" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column4" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column5" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column6" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column7" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column8" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+      </div>\
+      <div class = "row" id="grid_7">\
+        <div id="column1" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column2" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column3" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column4" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column5" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column6" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column7" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column8" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+      </div>\
+      <div class = "row" id="grid_8">\
+        <div id="column1" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column2" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column3" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column4" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column5" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column6" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column7" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
+        <div id="column8" class="grid-box col-xs-2" myAttr="canMove" combine="canCombine">0</div>\
```

Comment:
Whenever you have to copy/paste more than 10 times, or things that are the same (except for a few variables) for more 10 lines, it's usually an indicator that you can do it a lot more elegantly.